### PR TITLE
Fix *.modules.scss that don't get correctly typed

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

This was only a problem within vscode (and maybe other editors) which is why I marked it as a `chore`.

### CODE REVIEW

File: types/mocks.d.ts [hunk 1]
```diff
@@ -10,14 +10,6 @@ declare module "@hapi/subtext"
 
 // Support import for custom module extensions
 // https://www.typescriptlang.org/docs/handbook/modules.html#wildcard-module-declarations
-declare module "*.module.scss" {
```
Comment (Nokel81) [inline types/mocks.d.ts]: Since we have that package which types them we no longer need these.

**Review by aleksfront — APPROVED** [review]
LGTM

**Review by ixrock — APPROVED** [review]
Why only these files? Looks like a hack.

**Comment by Nokel81** [comment]
@ixrock I looked at the typescript server logs in vscode to figure out why a specific file was being typed as `{}` instead of being typed as `{ AddCluster: string; }`. It reported an error from the CSS modules plugin for typescript saying that the `theme-light` mixin was undefined and that it considered the given file the "root file" for that use.

So I figured that importing the file that defines the `theme-light` mixin would help solve the issue. 

I also tried getting the plugin to accept the mixin file as "always imported" but I couldn't figure out how. I also don't know how the actual transpiler does it.

**Pull request merged**